dizoo/common/policy/md_rainbow_dqn.py

```diff
@@ -70,9 +92,9 @@ def _forward_learn(self, data: dict) -> Dict[str, Any]:
         # =============
         # after update
         # =============
-        self._armor.target_update(self._armor.state_dict()['model'])
+        self._target_model.update(self._learn_model.state_dict())
         return {
             'cur_lr': self._optimizer.defaults['lr'],
             'total_loss': loss.item(),
-            'priority': td_error_per_sample_mean.abs().tolist(),
+            'priority': td_error_per_sample.abs().tolist(),
```

Comment:
You need use `td_error_per_sample_mean` as new priority value, for multi-discrete action space, there are multiple `td_error_per_sample` and you need to average them. And for compatibility of naive discrete action space, you can add `td_error_per_sample_mean = td_error_per_sample` in corresponding code block.

Reply:
fixed